<!-- <?xml version="1.0"?> -->
<launch>
	<group ns = "bag_handle_estimator">

		<param name="execute_default" type="bool" value="true"/>
		<param name="pub_plane_cloud" type="bool" value="true"/>
	
		<param name="sub_ctrl_topic_name" type="str" value="execute_ctrl"/>

		<!-- D405 only -->
		<!-- <param name="sub_point_topic_name" type="str" value="/camera/depth_registered/points"/> -->
		<!-- <param name="base_frame_name" type="str" value="camera_link"/> -->
		<!-- <param name="camera_frame_name" type="str" value="camera_depth_frame"/> -->
		<!-- <param name="map_frame_name" type="str" value="camera_link"/> -->

		<!-- SOBIT PRO -->
		<param name="sub_point_topic_name" type="str" value="/hand_camera/depth_registered/points"/>
		<param name="base_frame_name" type="str" value="hand_camera_link"/>
		<param name="camera_frame_name" type="str" value="hand_camera_depth_frame"/>
		<param name="map_frame_name" type="str" value="arm_base_link"/>

		<!-- depth range -->
		<param name="depth_range_min_x" type="double" value="0.0"/>
		<param name="depth_range_max_x" type="double" value="0.5"/>
		<!-- height range -->
		<param name="depth_range_min_z" type="double" value="-0.5"/>
		<param name="depth_range_max_z" type="double" value="1.0"/>
		
		<node pkg="bag_handle_estimator" type="bag_handle_estimator" name="bag_handle_estimator_node" output="screen"/>

	</group>

</launch>
